src/app/upload_services/BaseUploadService.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC, abstractmethod
+import importlib
+import os
+import re
+
+from werkzeug.datastructures import FileStorage
+
+
+class BaseUploadService(ABC):
+    def __init__(self, datatype: str):
+        self.EXT_REGEX = os.environ.get("EXT_REGEX", "json")
+        self.datatype = datatype
+        self.encryption_enabled = False
+        pass
+
+    def verify(self, filename: str):
+        return re.search(self.EXT_REGEX, filename)

```

Comment:
We might want to match specifically the extension at the end. Otherwise `this_is_not_json.exe` will match.

Also might want to consider casting that to a boolean. This will work and can be used as a boolean as is but I feel like it's a bit unintuitive that this is returning a match object rather than a boolean. Even though you can use the search response as a bool without any problems

Reply:
added cast to boolean